fabric-rendering-v1/src/testmod/java/net/fabricmc/fabric/test/rendering/client/ArmorRenderingTests.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright (c) 2016, 2017, 2018, 2019 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.fabricmc.fabric.test.rendering.client;
+
+import net.minecraft.client.MinecraftClient;
+import net.minecraft.client.render.entity.model.BipedEntityModel;
+import net.minecraft.client.render.entity.model.EntityModelLayers;
+import net.minecraft.entity.EquipmentSlot;
+import net.minecraft.entity.LivingEntity;
+import net.minecraft.item.Items;
+import net.minecraft.util.Identifier;
+
+import net.fabricmc.api.ClientModInitializer;
+import net.fabricmc.fabric.api.client.rendering.v1.ArmorRenderer;
+
+public class ArmorRenderingTests implements ClientModInitializer {
+	private BipedEntityModel<LivingEntity> armorModel;
+	private Identifier texture = new Identifier("textures/block/dirt.png");
+
+	@Override
+	public void onInitializeClient() {
+		ArmorRenderer.register((matrices, vertexConsumers, stack, entity, slot, light, model) -> {
```

Comment:
Please add a few words what this does/what it's supposed to change/look like so someone porting the code can quickly verify correct behavior without guessing from the code.